components/org.wso2.carbon.identity.local.auth.smsotp.provider/src/main/java/org/wso2/carbon/identity/local/auth/smsotp/provider/impl/CustomProvider.java

```diff
@@ -62,23 +61,22 @@ public void send(SMSData smsData, SMSSenderDTO smsSenderDTO, String tenantDomain
                     + " using custom provider");
         }
 
-        SMSMetadata smsMetadata = new SMSMetadata();
+        if (StringUtils.isBlank(smsSenderDTO.getContentType())
+                || smsSenderDTO.getContentType().equals(Constants.JSON)) {
+            smsData.setContentType(Constants.APPLICATION_JSON);
+        } else if (smsSenderDTO.getContentType().equals(Constants.FORM)) {
```

Comment:
swap the constant and the getter

Reply:
Fixed with https://github.com/wso2-extensions/identity-local-auth-smsotp/pull/19/commits/7f8ecec500c660c6c90bc50c9c65a3656e5b8972